### PR TITLE
Update sl-graphql.md

### PR DESCRIPTION
**Pull request by mirnawong1**
clarify schema explorer urls for other regions:

- [North America multi-tenant](https://semantic-layer.cloud.getdbt.com/api/graphql)
- [EMEA multi-tenant](https://semantic-layer.emea.dbt.com/api/graphql)
- [APAC multi-tenant](https://semantic-layer.au.dbt.com/api/graphql)
- North America Cell 1 &mdash; `https://semantic-layer.{account prefix}.us1.dbt.com/api/graphql`
- ST

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 1]
```diff
@@ -22,7 +22,20 @@ GraphQL has several advantages, such as self-documenting, having a strong typing
...
+
+The schema explorer URLs vary depending on your [deployment region](/docs/cloud/about-cloud/regions-ip-addresses). Use the following table to find the right link for your region:
+
+| Deployment type |	Schema explorer URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://semantic-layer.cloud.getdbt.com/api/graphql |
+| EMEA multi-tenant	|	https://semantic-layer.emea.dbt.com/api/graphql |
+| APAC multi-tenant	|	https://semantic-layer.au.dbt.com/api/graphql |
+| Single tenant | `https://semantic-layer.YOUR_ACCESS_URL.com/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
+| North America Cell 1	|	`https://semantic-layer.YOUR_ACCOUNT_PREFIX.us1.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
```
Comment (DevonFulcher) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]: I believe this should be `https://YOUR_ACCOUNT_PREFIX.semantic-layer.us1.dbt.com/api/graphql`. Also, I'm unsure if the ST one should also be flipped.

Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
| North America Cell 1	|	`https://YOUR_ACCOUNT_PREFIX.semantic-layer.us1.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
```

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 2]
```diff
@@ -22,7 +22,20 @@ GraphQL has several advantages, such as self-documenting, having a strong typing
...
+The dbt Semantic Layer GraphQL API allows you to explore and query metrics and dimensions. Due to its self-documenting nature, you can explore the calls conveniently through a schema explorer. 
+
+The schema explorer URLs vary depending on your [deployment region](/docs/cloud/about-cloud/regions-ip-addresses). Use the following table to find the right link for your region:
+
+| Deployment type |	Schema explorer URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://semantic-layer.cloud.getdbt.com/api/graphql |
+| EMEA multi-tenant	|	https://semantic-layer.emea.dbt.com/api/graphql |
+| APAC multi-tenant	|	https://semantic-layer.au.dbt.com/api/graphql |
+| Single tenant | `https://semantic-layer.YOUR_ACCESS_URL.com/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
| Single tenant | `https://YOUR_ACCESS_URL.semantic-layer/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
```

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 3]
```diff
@@ -22,7 +22,20 @@ GraphQL has several advantages, such as self-documenting, having a strong typing
...
+| Deployment type |	Schema explorer URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://semantic-layer.cloud.getdbt.com/api/graphql |
+| EMEA multi-tenant	|	https://semantic-layer.emea.dbt.com/api/graphql |
+| APAC multi-tenant	|	https://semantic-layer.au.dbt.com/api/graphql |
+| Single tenant | `https://YOUR_ACCESS_URL.semantic-layer/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
+| North America Cell 1	|	`https://YOUR_ACCOUNT_PREFIX.semantic-layer.us1.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
+
+**Example**
+- If your access URL is `ABC123.getdbt.com`, your schema explorer URL will be `https://semantic-layer.ABC123.getdbt.com/api/graphql`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
- If your access URL is `ABC123.getdbt.com`, your schema explorer URL will be `https://ABC123.getdbt.com.semantic-layer/api/graphql`.
```

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-cloud-apis/sl-graphql.md]: thank you @DevonFulcher ! i've changed this now. it's different to the regional MT ones where `semantic-layer` comes before the URL? 

i don't have a ST instance to test but is the updated PR correct?

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 4]
```diff
@@ -22,7 +22,20 @@ GraphQL has several advantages, such as self-documenting, having a strong typing
...
+
+The schema explorer URLs vary depending on your [deployment region](/docs/cloud/about-cloud/regions-ip-addresses). Use the following table to find the right link for your region:
+
+| Deployment type |	Schema explorer URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://semantic-layer.cloud.getdbt.com/api/graphql |
+| EMEA multi-tenant	|	https://semantic-layer.emea.dbt.com/api/graphql |
+| APAC multi-tenant	|	https://semantic-layer.au.dbt.com/api/graphql |
+| Single tenant | `https://YOUR_ACCESS_URL.semantic-layer/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate Access URL for your region and plan.|
+| North America Cell 1	|	`https://YOUR_ACCOUNT_PREFIX.semantic-layer.us1.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
| North America Cell 1	| `https://YOUR_ACCOUNT_PREFIX.semantic-layer.<region>.dbt.com/api/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account prefix and make sure you leave `us1.dbt.com` in the URL. |<br />
```

**Comment by mirnawong1** [comment]
hey @DevonFulcher , ok have made changes based on what Kyle mentioned. please take a look and let me know if this is looking good? thank you!

**Review by DevonFulcher — APPROVED** [review]
Thank you!

**Pull request merged**